### PR TITLE
Show exam information on book view

### PR DESCRIPTION
**Pull request by felipecalvo**
## :dart: Goal

Show course, start and end date, and available time for an exam on book view.

## :memo: Details

Course is only visible for teachers.

I'm also removing the indexing (`1. Examen`) on exam listing as students should see only one exam at a time, and while teachers could see many, they don't have a natural order anyway.

### Student view
image

### Teacher view
image

## :warning: Dependencies

https://github.com/mumuki/mumuki-domain/pull/245, as that PR allows teachers to access exams through navigation.

## :back: Backwards compatibility

Yes

## :soon: Future work

None

### CODE REVIEW

File: app/helpers/exams_helper.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module ExamsHelper
+  def exam_information_for(user, exam)
+    %Q{
+      #{if user.teacher_here?
+          "#{fa_icon('graduation-cap', class: 'fa-fw',
+                     text: "<strong>#{t :course}:</strong> #{exam.course.canonical_code}".html_safe)}
+          <br>"
+        end
+      }
```
Comment (fedescarpa) [inline app/helpers/exams_helper.rb:9]: Could you estract this logic to another method?

**Review by fedescarpa — COMMENTED** [review]
LGTM bu please, take a look to my comment :smile:

File: app/helpers/exams_helper.rb (hunk 1 above)
Comment (felipecalvo) [reply app/helpers/exams_helper.rb:9]: Sure!

**Comment by felipecalvo** [comment]
I rebased that last commit with the proposed change by @fedescarpa so history is cleaner.

Merging once checks pass.

**Pull request merged**